config_loader.py

```diff
@@ -0,0 +1,72 @@
+import yaml
+
+'''
+Configuration keys enumeration
+'''
+
+
+class ConfigKeys:
+    DESKS_ARRAY = "desks"
+    DESK_ID = "id"
+    DESK_NAME = "name"
+    DESK_SENSOR_PIN = "sensor_pin"
+    DEBUG_MODE = "debug_mode"
+
+
+''' 
+Singleton configuration instance
+'''
+__config = None
```

Comment:
Don't need this.

Reply:
Why not? It needs to be defined at the modular level since it is static and does not belong to any object instance, so this is the only place we can define it ?

Reply:
It can be created directly in `get_global_config` without being defined outside first.